app/resources/forms.js.template

```diff
@@ -368,12 +369,18 @@ function make_form_expandable(expandedSection) {
   });
 }
 
-function form_confirmation() {
-   var personName = $('given_name').value + ' ' + $('family_name').value;
-   if (confirm('Do you want to create a record for ' + personName + '?\n' +
-               'Press OK to submit the form.\n' +
-               'Press Cancel to return to the form.'))
-      return validate_fields();
-   else
-      return false;
+// Ask for confirmation when submitting a form
+// familyNameFirst: true or false
+function confirm_submit(familyNameFirst) {
+  var personName = $('given_name').value + ' ' + $('family_name').value;
+
+  if (familyNameFirst)
```

Comment:
It would be easier to read if you use:
```js
var personName = familyNameFirst ? ... : ...;
```
or
```js
var personName;
if (familyNameFirst)
  ...
else
  ...
```